### PR TITLE
Removing the images that are no longer being managed in azure-pipelin…

### PR DESCRIPTION
**Pull request by shadyysf**
…es-image-generation and live in the new virutal-evnironments repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+THIS NEEDS TO BE REVISITED, MENTION THIS IN PR.
```
Comment (shadyysf) [inline README.md]: @kaylangan , I need your help with the wording on this specific file. Most things in this file are not valid anymore.

Comment (shadyysf) [reply README.md]: I took a stab at that readme file, I'll go ahead and merge this PR but let me know if you think this file needs further changes.

File: images/linux/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Azure pipelines Linux-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. For future contributions or issues please use the virtual-environments repository instead. You can still view the commit history of those images from when they used to live in the azure-pipelines-image-generation repository.
```
Comment (kaylangan) [inline images/linux/README.md]:
```suggestion
Azure Pipelines Linux-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. Please submit issues and pull requests in the virtual-environments repository instead. You can still view the full commit history of the images in this repository.
```

File: images/macos/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The README.md files for the Azure pipelines Mac-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. For future contributions or for issues regarding any of the images, please use the virtual-environments repository instead. You can still view the commit history of those images from when they used to live in the azure-pipelines-image-generation repository.
```
Comment (kaylangan) [inline images/macos/README.md]:
```suggestion
The README.md files for the Azure Pipelines Mac-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. Please submit issues in the virtual-environments repository instead.
```

File: images/win/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+The VS2019 and VS2017 Azure pipelines Windows-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. The Windows Container image is still being managed in the azure-pipelines-image-generation repository for now. For future contributions to the VS2017 or VS2019 image or for issues regarding any of the images, please use the virtual-environments repository instead. You can still view the commit history of those images from when they used to live in the azure-pipelines-image-generation repository.
```
Comment (kaylangan) [inline images/win/README.md]:
```suggestion
The VS2017 and VS2019 Azure Pipelines Windows-based images are now in the [virtual-environments](https://github.com/actions/virtual-environments) repository. The Windows Container image is still being managed in this repository. Please submit issues and pull requests for the VS2017 and VS2019 images in the virtual-environments repository instead. You can still view the full commit history of the images in this repository.
```

File: README.md [hunk 5]
```diff
@@ -1,57 +1,16 @@
...
-### Linux
-
-- [How to use Packer to create Linux VM images in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/build-image-with-packer)
-
-### Windows
-
-- [Instructions to build Windows images](/help/CreateImageAndAzureResources.md)
-- [How to use Packer to create Windows VM images in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/build-image-with-packer)
-
-## Contributing
```
Comment (kaylangan) [inline README.md]: Let's keep this section

File: README.md [hunk 6]
```diff
@@ -1,57 +1,16 @@
...
 
-If you have an issue or request for an image, please create an issue in this repo. To add or change a tool yourself, see [Contributing](#contributing).
-
-## Image generation help
-
-### Linux
-
-- [How to use Packer to create Linux VM images in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/build-image-with-packer)
-
-### Windows
```
Comment (kaylangan) [inline README.md]: Should we keep this as it applies to WINCON?

File: README.md [hunk 7]
```diff
@@ -1,57 +1,16 @@
...
+This repo contains scripts that generate the Windows Container virtual machine images used by [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/?nav=min) for [Microsoft-hosted CI/CD](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=vsts&tabs=yaml).The images are generated using [Packer](https://www.packer.io/).
 
 ## Following image updates
-You can follow releases [here](https://github.com/Microsoft/azure-pipelines-image-generation/releases). Consider enabling **Watch -> Releases only** at the top of the page.
-
-## Notifications
-Subscribe to [this RSS feed](https://microsoft.github.io/azure-pipelines-image-generation/) to be notified of upcoming image changes and other important messages. General Azure Pipelines service health is found [here](https://status.dev.azure.com/).
-
-# Image contents
```
Comment (kaylangan) [inline README.md]: Should we keep this for WINCON?

File: README.md (hunk 6 above)
Comment (shadyysf) [reply README.md]: Makes sense. I added it back.

**Pull request merged**